src/contracts/Protocol/Vault.sol

```diff
@@ -124,6 +124,10 @@ contract Vault is Ownable, OnlyCostController {
         depositAccounting(user, amount, questionId, stage);
     }
 
+    function burnMetric(address user, uint256 amount) external onlyCostController {
+        metric.transferFrom(user, treasury, amount);
```

Comment:
so the treasury is just some address we what is the address are we truly burning these tokens?